ubi10-openjdk-21.yaml

```diff
@@ -51,7 +51,7 @@ modules:
   - name: jboss.container.openjdk.jdk
     version: "21"
   - name: jboss.container.maven
-    version: "3.8.21"
+    version: "3.9.21"
```

Comment:
Check your version 3.9.21 does not exist. It looks like the upstream head is 3.9.9

Reply:
3.9.21 is the version of the cekit module (not the maven version specifically). It's used to select the right version when more than one is available, e.g. in the ubi9 branch you can see `modules/maven/11`, `modules/maven/17` and ``modules/maven/21`. They all have the same module name, but different versions.

The scheme is `<maven  major version>.<maven minor version>.JDK version`.

Reply:
Belt and braces, the module and the image yaml match:
```
$ git grep 3.9.21
modules/maven/21/module.yaml:version: '3.9.21'
ubi10-openjdk-21.yaml:    version: "3.9.21"
```